scalopus_tracing/src/native/endpoint_native_trace_sender.cpp

```diff
@@ -62,16 +62,11 @@ EndpointNativeTraceSender::~EndpointNativeTraceSender()
 static Data process_events(const EventMap& tid_event_map)
 {
   std::map<std::string, cbor::cbor_object> my_data;
-  my_data["pid"] = cbor::cbor_object::make(static_cast<unsigned long>(::getpid()));
+  my_data["pid"] = cbor::cbor_object{static_cast<unsigned long>(::getpid())};
+  my_data["events"] = cbor::cbor_object{tid_event_map};
 
-  std::vector<std::tuple<EventMap::key_type, EventMap::mapped_type>> event_entries;
```

Comment:
We don't need to read as a vector of tuples anymore, because the new `cbor` library properly supports maps, instead of making a vector of pairs.

Reply:
Could you also do? :
``` c++
const std::map<std::string, cbor::cbor_object> my_data
{
  { "pid", cbor::cbor_object{ static_cast<unsigned long>(::getpid()) } },
  { "events", cbor::cbor_object{ tid_event_map } }
};
```